LabApiUtilities/src/main/com/microsoft/identity/labapi/utilities/client/LabClient.java

```diff
@@ -184,6 +188,19 @@ private String getPassword(final String credentialVaultKeyName) throws LabApiExc
         return getSecret(secretName);
     }
 
+    public Boolean resetPassword(@NonNull final String upn) {
+        ResetApi resetApi = new ResetApi();
+        try {
+            final CustomSuccessResponse resetResponse = resetApi.apiResetPut(upn, ResetOperation.PASSWORD.toString());
+            Thread.sleep(PASSWORD_RESET_WAIT_DURATION);
+            final String expectedResult = ("Password reset successful for user : " + upn)
+                    .toLowerCase();
+            return resetResponse.getResult().toLowerCase().contains(expectedResult);
+        } catch (ApiException | InterruptedException e) {
+            throw new RuntimeException("Error resetting lab user password", e);
```

Comment:
Let's throw a `LabApiException` here as we've been trying to use checked exceptions here in the LabApiUtiltities